Makefile

```diff
@@ -6,11 +6,15 @@ VERSION ?= $(error VERSION environment variable must be set)
 
 # Run cargo fmt
 rustfmt: deps-build
-	cargo fmt -- --check
+	cargo fmt --all --check
+
+# NOTE: Check all targets, then check the build target specifically. Otherwise, it might build for
+# testing but not otherwise due to feature resolution shenanigans.
```

Comment:
note says checks all targets, but not all targets are being checked

Reply:
That's what the --all-targets line below does.

Reply:
sry misread the diff